platforms/core-configuration/model-core/src/main/java/org/gradle/internal/extensibility/package-info.java

```diff
@@ -13,7 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@NonNullApi
-package org.gradle.internal.extensibility;
 
-import org.gradle.api.NonNullApi;
+/**
+ * Package {@code org.gradle.internal.extensibility}.
```

Comment:
I don't think this comment is very helpful.

Reply:
Hm, in the original big-boom PR, I saw some codestyle errors that required adding a javadoc to a package. Maybe I got confused. I removed this javadoc now and at least `sanityCheck` passes locally.